### PR TITLE
Stack and queue

### PR DESCRIPTION
**Pull request by rnmessick**
- [x] Top-level README “Table of Contents” is updated
 - [X ] Feature tasks for this challenge are completed
 - [ X] Unit tests written and passing
     - [X ] “Happy Path” - Expected outcome
     - [ X] Expected failure
     - [ X] Edge Case (if applicable/obvious)
 - [ X] README for this challenge is complete
     - [X ] Summary, Description, Approach & Efficiency, Solution
     - [ X] Link to code
     - [ X] Picture of whiteboard

### CODE REVIEW

File: code401challenges/src/main/java/code401challenges/stacksandqueues/PseudoQueue.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    public PseudoQueue() {
+        this.stackOne = new Stack<>();
+        this.stackTwo = new Stack<>();
+    }
+
+    public void enqueue(T value) {
+        this.stackOne.push(value);
+    }
+
+    public T dequeue() {
```
Comment (evanslaton) [inline code401challenges/src/main/java/code401challenges/stacksandqueues/PseudoQueue.java]: This method doesn't quite work in all scenarios. If you have a `pseudoqueue` and you `enqueue()` `a`, `b` and `c`:
```
     c
     b
     a
firstStack      secondStack
```
Then we call `dequeue()` and we should get back `a` since `a` was the first value put into the `pseudoqueue`. This code will move `c` into `secondStack` and then return `b`.
```
     
     b <--
     a                      c
firstStack      secondStack
```
